### PR TITLE
Version 1.0.3

### PR DESCRIPTION
**Pull request by cosminbosutar**
Improved the 'form' template.
Personalized the orders identifier to make it more framework specific.
Ensured that the "orderId" sent inside the request payment is always unique.
Updated the readme file.
Fixed the display of the administrator email address in case of payment failure.
Added check that excludes orders with the amount below or equal to zero.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,98 @@
...
+- Safe payments – peace of mind while paying online
+- Instant billing and receipts – faster shipments and delivery
+- Smooth purchase flow – straightforward shopping experience
+
+We take pride in offering world class, free customer support to all our merchants during the integration phase, and at any time thereafter. Our support team is available non-stop during regular business hours EET.
+
+## Installation
+The easiest way of installing our module is by visiting the [official module page](https://marketplace.magento.com/twispay-twispay-tpay.html).
+Alternatively, you can check out our [installation guide](#) for detailed step by step instructions.
+1. Login to the official Magento webssite and [login](https://account.magento.com/customer/account/login)
```
Comment (dragos-ursu) [inline README.md]: Please fix typo `webssite`.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,98 @@
...
+3. Search for the **twispay** keyword
+4. Open [Twispay Credit Card Payments](https://marketplace.magento.com/twispay-twispay-tpay.html) for **Magento 1**
+5. Select you **Magento 1 version** and add it to cart
+6. Go to checkout and click on **Place Order**
+7. Click on **Install**
+8. Copy the **Access Key** by clicking the **Copy** button from the **Action** column
+9. Login to your **Magento 1** admin panel
+10. Go to **System** -> **Magento Connect** -> **Magento Connect Manager**
+11. Re-enter you **administrator credentials** and login
+12. Paste the copied **Access Key** inside the **Paste extention key to install** and click on the **Install** button
```
Comment (dragos-ursu) [inline README.md]: Please fix typo `extention`.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,98 @@
...
+13. Wait for the installation to finish
+14. Go back to your **Magento 1** admin panel by clicking on the **Return to Admin** link
+15. Go to **System** -> **Configuration**
+16. Go to **Payment Methods**
+17. Select **Yes** under **Live mode**. _(Unless you are testing)_
+18. Enter your **Site ID**. _(You can get one from [here for live](https://merchant.twispay.com/login) or from [here for stage](https://merchant-stage.twispay.com/login))_
+19. Enter your **Private Key**. _(You can get one from [here for live](https://merchant.twispay.com/login) or from [here for stage](https://merchant-stage.twispay.com/login))_
+20. Enter your tehnical **Contact Email**. _(This will be displayed to customers in case of a payment error)_
```
Comment (dragos-ursu) [inline README.md]: Please fix typo `tehnical`.

File: package.xml [hunk 4]
```diff
@@ -21,10 +21,16 @@ Added functionality to create a transaction after each successful payment.&#xD;
 Added purchase order type refund and partial refund support.&#xD;
 Added recurring order type refund and partial refund support.&#xD;
 Added daily sync with the Twispay server for recurring profiles.</notes>
```
Comment (dragos-ursu) [inline package.xml]: Please fix wrong closing `</notes>`.

File: app/locale/en_US/Twispay_Tpay.csv [hunk 5]
```diff
@@ -54,3 +54,6 @@
 " No transaction found."," No transaction found."
 " Removed redundant order witd ID: "," Removed redundant order witd ID: "
```
Comment (dragos-ursu) [inline app/locale/en_US/Twispay_Tpay.csv]: Please fix typo `witd`.

File: app/locale/en_US/Twispay_Tpay.csv [hunk 6]
```diff
@@ -54,3 +54,6 @@
 " No transaction found."," No transaction found."
 " Removed redundant order witd ID: "," Removed redundant order witd ID: "
 " Payment for order #%s canceled as free trial perios are not allowed."," Payment for order #%s canceled as free trial perios are not allowed."
```
Comment (dragos-ursu) [inline app/locale/en_US/Twispay_Tpay.csv]: Please fix typo `perios`.

**Review by dragos-ursu — CHANGES_REQUESTED** [review]
Please fix the few typos found.

File: README.md (hunk 1 above)
Comment (cosminbosutar) [reply README.md]: Solved

File: README.md (hunk 2 above)
Comment (cosminbosutar) [reply README.md]: Solved

File: README.md (hunk 3 above)
Comment (cosminbosutar) [reply README.md]: Solved

File: package.xml (hunk 4 above)
Comment (cosminbosutar) [reply package.xml]: Solved

File: app/locale/en_US/Twispay_Tpay.csv (hunk 5 above)
Comment (cosminbosutar) [reply app/locale/en_US/Twispay_Tpay.csv]: Solved

File: app/locale/en_US/Twispay_Tpay.csv (hunk 6 above)
Comment (cosminbosutar) [reply app/locale/en_US/Twispay_Tpay.csv]: Solved

**Comment by dragos-ursu** [comment]
Code review done.

**Review by dragos-ursu — APPROVED** [review]
Code review done.

**Pull request merged**